src/AccountActionsHistory.js

```diff
@@ -237,58 +250,109 @@ class AccountActionsHistory extends React.Component {
 
           <Row>
             <Col xs={1.5}>
-              <KeyValue label={this.props.stripes.intl.formatMessage({ id: 'ui-users.details.field.feetype' })} value={_.get(account, ['feeFineType'], '-')} />
+              <KeyValue
+                label={<FormattedMessage id="ui-users.details.field.feetype" />}
+                value={_.get(account, ['feeFineType'], '-')}
```

Comment:
maybe it makes sense to import only `get` from `lodash/get`

Reply:
@aliaksei-chumakou I think it is not an issue anymore. @ttamj, could you please confirm that `wepback` plugin will be added to tree-shake `lodash` dependencies?

Reply:
@VictorSoroka1, yes that is correct.